tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -134,6 +148,11 @@ trait VirtualScalaJSIRFile extends VirtualFile {
   def infoAndTree: (ir.Infos.ClassInfo, ir.Trees.ClassDef)
 }
 
+trait VirtualRelativeIRFile extends VirtualScalaJSIRFile
```

Comment:
Similarly to `IRContainer`, this should be called `VirtualRelativeScalaJSIRFile`.